test/resources/prometheus.py

```diff
@@ -16,10 +16,21 @@ def check_prometheus_query(host: str, port: int, query: str) -> None:
         raise Exception(f"Prometheus query failed with status code {response.status_code}")
     if response.json().get("status") != "success":
         raise Exception(f"Prometheus query failed with status: {response.json().get('status')}")
+    return response
+
+
+def check_prometheus_query(host: str, port: int, query: str) -> None:
+    response = _run_prometheus_query(host, port, query)
     if not response.json().get("data", {}).get("result") or len(response.json().get("data", {}).get("result")) == 0:
         raise Exception("Prometheus query returned no results")
 
 
+def check_prometheus_query_is_missing(host: str, port: int, query: str) -> None:
+    response = _run_prometheus_query(host, port, query)
+    if response.json().get("data", {}).get("result") and len(response.json().get("data", {}).get("result")) == 1:
```

Comment:
Is the result always 1, or should the exception be raised if `!=0`?

Reply:
fixed in https://github.com/openshift/microshift/pull/4769/commits/b2563d6a8d2d84ae0d2d9b6e433c20e630e46faa commit